app/src/androidTest/java/org/mozilla/focus/activity/ClearCacheTest.java

```diff
@@ -0,0 +1,66 @@
+package org.mozilla.focus.activity;
+
+import android.content.Intent;
+import android.support.annotation.Keep;
+import android.support.test.rule.ActivityTestRule;
+import android.support.test.runner.AndroidJUnit4;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mozilla.focus.R;
+import org.mozilla.focus.utils.AndroidTestUtils;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.matcher.RootMatchers.withDecorView;
+import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.not;
+
+@Keep
+@RunWith(AndroidJUnit4.class)
+public class ClearCacheTest {
+
+    // Defer the startup of the activity cause we want to avoid First Run / Share App / Rate App dialogs
+    @Rule
+    public final ActivityTestRule<MainActivity> activityRule = new ActivityTestRule<>(MainActivity.class, true, false);
+
+    @Before
+    public void setUp() {
+        // Set the share preferences and start the activity
+        AndroidTestUtils.beforeTest();
```

Comment:
Use Builder like this
https://github.com/mozilla-tw/FirefoxLite/blob/6ad7efce888c6c1bd3a6a4bec785ab1ab2a1fce4/app/src/androidTest/java/org/mozilla/focus/activity/WebContextMenuTest.java#L117
We should deprecate beforeTest soon

Reply:
applying builder